src/test/snapshot-controller/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+
+appVersion: 2.1.1
+name: snapshot-controller
+description: A Helm chart for snapshot-controller
+version: 2.1.1
+kubeVersion: ">=1.17.0-0"
+home: https://github.com/kubernetes-csi/external-snapshotter
+sources:
+  - https://github.com/kubernetes-csi/external-snapshotter
```

Comment:
Should be this repo, i.e. https://github.com/kubesphere/helm-charts

Reply:
OK